plugins/wporg-5ftf/includes/email.php

```diff
@@ -22,7 +23,7 @@
  *
  * @return bool
  */
-function send_email( $to, $subject, $message, $pledge_id ) {
+function send_email( $to, $subject, $message, $pledge_id = false ) {
 	$headers = array(
 		'From: WordPress - Five for the Future <donotreply@wordpress.org>',
```

Comment:
That's a good point. I was thinking it was used in order to match the `Return-Path`, so that spam filters don't think we're forging `From`. I just checked though, and the `Return-Path` is `bounce@wordpress.org`.

We could change it to that. It technically has the same meaning, but most folks won't be familiar with the term, so it's not as bad.

I'm leery of changing the `From` to `getinvolved@` though, because of those spam filters.

Reply:
I set it to `bounce@` in 7cad4bd, but I'm open to other ideas if you think there's a better way.